src/main/java/servlets/ResultServlet.java

```diff
@@ -0,0 +1,135 @@
+package servlets;
+
+import java.io.IOException;
+import java.io.PrintWriter;
+import java.util.*;
+import javax.servlet.annotation.WebServlet;
+import javax.servlet.http.HttpServlet;
+import javax.servlet.http.HttpServletRequest;
+import javax.servlet.http.HttpServletResponse;
+import javax.servlet.http.HttpSession;
+import helpers.SomeHelper;
+
+@WebServlet(
+        name = "ResultServlet",
+        urlPatterns = "/calc/result"
+)
+public class ResultServlet extends HttpServlet {
+
+    public static String toPostfix(String infix) {
+        char temp;
+        StringBuilder result = new StringBuilder();
+        StringBuilder hlp = new StringBuilder();
+
+        for (int i = 0; i < infix.length(); i++) {
+            if ('(' == infix.charAt(i)) {
+                hlp.append(infix.charAt(i));
+            } else if (')' == infix.charAt(i)) {
+                temp = hlp.substring(hlp.length() - 1).charAt(0);
+                while ('(' != temp) {
+                    result.append(" ").append(temp);
+                    hlp.setLength(hlp.length() - 1);
+                    temp = hlp.substring(hlp.length() - 1).charAt(0);
+                }
+                hlp.setLength(hlp.length() - 1);
+            } else if (SomeHelper.isOperator(infix.charAt(i))) {
+                while (hlp.length() != 0) {
+                    temp = hlp.substring(hlp.length() - 1).charAt(0);
+                    if (SomeHelper.isOperator(temp) && (SomeHelper.precedence(infix.charAt(i)) <= SomeHelper.precedence(temp))) {
+                        result.append(" " + temp + " ");
+                        hlp.setLength(hlp.length() - 1);
+                    } else {
+                        break;
+                    }
+                }
+                result.append(" ");
+                hlp.append(infix.charAt(i));
+            } else
+                result.append(infix.charAt(i));
+        }
+        while (hlp.length() != 0) {
+            result.append(" " + hlp.substring(hlp.length() - 1));
+            hlp.setLength(hlp.length() - 1);
+        }
+
+        if (infix.charAt(0) == '-') {
+            int index = result.lastIndexOf("-");
+            result.deleteCharAt(index).deleteCharAt(index-1);
+            result.insert(1,'-');
+        }
+        System.out.println("!!!!" + result.toString() + "!!!!");
+        return result.toString();
+    }
+
+    public static int calculation(String st) {
+        int left, right;
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Use one line for each declaration, it enhances code readability.](https://app.codacy.com/manual/thejerome/IFMO_Servlets_20190915/pullRequest?prid=4887147)